libmscore/style.cpp

```diff
@@ -189,7 +189,7 @@ static const StyleType styleTypes[] {
       { Sid::measureSpacing,          "measureSpacing",          QVariant(1.2) },
       { Sid::staffLineWidth,          "staffLineWidth",          Spatium(0.08) },     // 0.09375
       { Sid::ledgerLineWidth,         "ledgerLineWidth",         Spatium(0.16) },     // 0.1875
-      { Sid::ledgerLineLength,        "ledgerLineLength",        Spatium(.6)   },     // notehead width + this value
+      { Sid::ledgerLineLength,        "ledgerLineLength",        Spatium(0.76) },     // notehead width + this value
```

Comment:
Add width to length to compensate for `Qt::FlatCap` not needed extra space, so by default the lines end up with the same length.